src/structurizr/model/model.py

```diff
@@ -106,20 +105,29 @@ def __init__(
         """
         super().__init__(**kwargs)
         self.enterprise = enterprise
-        self.people = set(people)
-        self.software_systems = set(software_systems)
         self.deployment_nodes = set(deployment_nodes)
         # TODO: simply iterate attributes
         self._elements_by_id = {}
         self._relationships_by_id = {}
         self._id_generator = SequentialIntegerIDGenerator()
 
     def __contains__(self, element: Element):
+        """Return True if the element is in the model."""
         return element in self.get_elements()
 
+    @property
+    def software_systems(self) -> Set[SoftwareSystem]:
+        """Return the software systems in the model."""
+        return set([e for e in self.get_elements() if isinstance(e, SoftwareSystem)])
```

Comment:
I'd suggest using a set comprehension:

```suggestion
        return {e for e in self.get_elements() if isinstance(e, SoftwareSystem)}
```

Reply:
I didn't even know you could do that!  Thanks.